src/module/actor/entity.js

```diff
@@ -14,7 +14,7 @@ const removeFalsyElements = (arr) =>
 
 export default class OseActor extends Actor {
   prepareDerivedData() {
-    this.system.prepareDerivedData?.();
+    if (game.version.startsWith("10")) this.system.prepareDerivedData?.();
```

Comment:
😭 

We're removing this before V100 right?

Reply:
God I hope so; I know people have a lingering distrust of core updates, but I absolutely hope that people aren't using v10 by time v100 rolls around